man/mysqlbinlog.1

```diff
@@ -1817,24 +1809,18 @@ shell> \fBmysqlbinlog \fR\fB\fIlog_file\fR\fR
 \&.\&.\&.
 # at 218
 #080828 15:03:08 server id 1  end_log_pos 258 	Write_rows: table id 17 flags: STMT_END_F
-BINLOG \'
-fAS3SBMBAAAALAAAANoAAAAAABEAAAAAAAAABHRlc3QAAXQAAwMPCgIUAAQ=
-fAS3SBcBAAAAKAAAAAIBAAAQABEAAAAAAAEAA//8AQAAAAVhcHBsZQ==
-\'/*!*/;
+BINLOG 'fAS3SBMBAAAALAAAANoAAAAAABEAAAAAAAAABHRlc3QAAXQAAwMPCgIUAAQ=
+fAS3SBcBAAAAKAAAAAIBAAAQABEAAAAAAAEAA//8AQAAAAVhcHBsZQ=='/*!*/;
```

Comment:
Is this wanted change? Does this work? Why it was on other line in original..

Reply:
Yes this works. The original had unnecessary use of backslash and new lines which I cleaned away.

![image](https://github.com/MariaDB/server/assets/668724/29233a0e-da09-40d2-8119-78d45c02dd5a)

Reply:
Again, actual man page re-flows the test automatically but the ASCII rendering will show the removed excess line changes.

![image](https://github.com/MariaDB/server/assets/668724/13d9c8e2-6f4c-4331-bff8-67f733ec0830)